app/View/Components/Menu.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\View\Components;
+
+use Closure;
+use Illuminate\Contracts\View\View;
+use Illuminate\View\Component;
+
+class Menu extends Component
+{
+    /**
+     * Create a new component instance.
+     */
+    public function __construct()
```

Comment:
puoi definirlo anche così, il costruttore può avere dei parametri:
ti lascio un esempio banale che però penso ti faccia capire
public $name;
    public $surname;
    public $birthday;
    /**
     * Create a new component instance.
     *
     * @return void
     */
    public function __construct($name, $surname, $birthday)
    {
        $this->name = $name;
        $this->surname = $surname;
        $this->birthday = $birthday;
    }

    /**
     * Get the view / contents that represent the component.
     *
     * @return \Illuminate\Contracts\View\View|\Closure|string
     */
    public function render()
    {
        return view('components.card');
    }

Reply:
corretto senza usare il costruct ma passando il parametro $classes al componente menu con le props